h2integrate/core/h2integrate_model.py

```diff
@@ -802,8 +821,30 @@ def connect_technologies(self):
                 self.plant.linear_solver = om.DirectSolver()
                 break
 
+        # initialize dispatch rules connection list
+        tech_to_dispatch_connections = self.plant_config.get("tech_to_dispatch_connections", [])
+
+        for connection in tech_to_dispatch_connections:
+            if len(connection) != 2:
+                err_msg = f"Invalid tech to dispatching_tech_name connection: {connection}"
+                raise ValueError(err_msg)
+
+            tech_name, dispatching_tech_name = connection
+
+            if tech_name == dispatching_tech_name:
+                continue
+            else:
+                # Connect the dispatch rules output to the dispatching_tech_name input
+                self.model.connect(
+                    f"{tech_name}.dispatch_block_rule_function",
+                    f"{dispatching_tech_name}.dispatch_block_rule_function_{tech_name}",
+                )
+
         if (pyxdsm is not None) and (len(technology_interconnections) > 0):
-            create_xdsm_from_config(self.plant_config)
+            try:
```

Comment:
@johnjasa  and @bayc I'd like your thoughts on if try, except here. I was having trouble running example 14 in Jupyter notebook due to a FileNotFoundError that only occurs in Jupyter. It works fine from the terminal (same environment). I decided that I don't think the XDSM diagram generation should stop execution and I put the try except around it with still printing the error message, but not as an error.